doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -82,175 +76,107 @@ To work with the nRF54H20 DK, follow the instructions in the next sections to in
+         #. Click **Install missing extensions**, then close VS Code.
+         #. Once the extensions are installed, click **Open VS Code** button again.
 
-         .. parsed-literal::
-            :class: highlight
+         You can then follow the instructions in :ref:`creating_vsc`.
 
-            /Users/*yourusername*/ncs-lcs/nrfutil toolchain-manager launch --shell --chdir "/Users/*yourusername*/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
+      .. tab:: Command line
 
-         .. note::
-            When working with west in the customer sampling release, you must always use a shell window with the |NCS| toolchain environment.
+         To build on the command line, complete the following steps:
 
-      #. Run the following commands in your terminal to install the correct lxml dependency:
+         1. With admin permissions enabled, download and install the `nRF Command Line Tools`_.
+         #. Restart the Toolchain Manager application.
+         #. Click the dropdown menu for the installed nRF Connect SDK version.
 
-         .. parsed-literal::
-            :class: highlight
+            .. figure:: ../../../../nrf/installation/images/gs-assistant_tm_dropdown.png
+               :alt: The Toolchain Manager dropdown menu for the installed nRF Connect SDK version, cropped
 
-            pip uninstall -y lxml
-            pip install lxml
+               The Toolchain Manager dropdown menu options
 
-      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+         #. Select :guilabel:`Open command prompt`.
 
-We recommend adding the nRF Util path to your environmental variables.
+         You can then follow the instructions in :ref:`creating_cmd`.
 
-.. _ug_nrf54h20_install_ncs:
-
-Installing the |NCS|
-====================
-
-After you have installed the toolchain, complete the following steps to get the customer sampling version of the |NCS|:
-
-1. In the terminal window opened during the toolchain installation, initialize west with the revision of the |NCS| from the customer sampling:
+Installing the Terminal application
+***********************************
 
-   .. parsed-literal::
-      :class: highlight
+On your computer, install `nRF Connect for Desktop`_.
+You must also install a terminal emulator, such as `nRF Connect Serial Terminal`_ (from the nRF Connect for Desktop application) or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).
 
-      west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.6.99-cs2
+Installing nRF Util and its commands
+************************************
 
-#. Enter the following command to clone the project repositories::
+Using the nRF54H20 DK with the |NCS| v2.7.0 requires the following:
 
-      west update
+* nRF Util version 7.11.1 or above
+* nRF Util ``device`` version 2.4.0
```

Comment:
Link here: https://nordic-dev.zoominsoftware.io/bundle/nrfutil/page/nrfutil-device/guides/programming.html

Reply:
I still think this link wouldn't be a bad idea.